docs/security.html

```diff
@@ -273,6 +275,24 @@
     sasl.kerberos.service.name="kafka"</pre>
             </li>
         </ol></li>
+
+    <li><h4><a id="security_rolling_upgrade" href="#security_rolling_upgrade">Incorporating Security Features in a Running Cluster</a></h4>
+        You can secure a running cluster via one or more of the supported protocols discussed previously. This is done in two phases:
+        <ul>
+            <li>Incrementally bounce the cluster nodes to open an additional, secured port.</li>
+            <li>Incrementally bounce again to close the plaintext port.</li>
+        </ul>
+        The steps for configuring SSL or SASL are described in sections <a href="#security_ssl">7.2</a> and <a href="#security_sasl">7.3</a>. Follow these steps to open an additional, secured port in Kafka. The plaintext port must be left open so brokers can communicate during the rolling bounce.
```

Comment:
And `to open an additional, secured port` should be consistent with whatever we do in the bullet point above.